packages/client/src/pages/RecommendationsPage/RecommendationsTable/RecommendationsTable.tsx

```diff
@@ -62,12 +68,53 @@ const getColumns = (useKilograms: boolean): GridColDef[] => [
   },
 ]
 
+function CustomPagination(
+  handleChange: (event: React.ChangeEvent<{ value: unknown }>) => void,
```

Comment:
This handleChange function can be used as a prop instead of a function parameter. And the declaration of this component can also match a shorthand declaration for consistency. Feel free to check out how RecommendationsTable is declared with its props on line 107!